modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -1,6 +1,17 @@
-= Getting Client ID and Client Secret of a registered Client App
+= Getting Client ID and Client Secret of a registered Client Application
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As Organization Administrator of the master organization, after you approve a client app to access your API, you can access information about the app from API Manager > Contracts. You can view and reset the unique client ID and client secret for the application. When calling the API, the client app owner needs to pass these credentials if the API is protected with policies.
\ No newline at end of file
+As a client application owner of a registered client, you might be required to provide client application credentials when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
```

Comment:
Typo in "the client secret information **for** the client application. "  for -> from